src/error.rs

```diff
@@ -88,14 +78,15 @@ pub enum Error {
     #[error("Invalid Amount Commitment")]
     AmountCommitmentInvalid,
 
+    #[error("Double spend detected")]
```

Comment:
hmm, I thought in our conversation this morning you said that Double Spend prevention is now a spentbook responsibility.  So I'm surprised to see this added...?